.github/workflows/ci-c.yml

```
@@ -21,22 +23,25 @@ jobs:
   tests:
     name: Unit Tests
     runs-on: 'ubuntu-latest'
-    container: greenbone/boreas-build:unstable
+    container: registry.community.greenbone.net/community/gvm-libs:edge
```

```suggestion
    container: ${{ vars.SELF_HOSTED_REGISTRY }}/community/gvm-libs:edge
```

I have introduced the variable, so it could be used!